chia/full_node/mempool_manager.py

```diff
@@ -389,50 +429,32 @@ async def add_spendbundle(
 
         if addition_amount > removal_amount:
-            return None, MempoolInclusionStatus.FAILED, Err.MINTING_COIN
+            return None, MempoolInclusionStatus.FAILED, Err.MINTING_COIN, None, []
 
         fees = uint64(removal_amount - addition_amount)
         assert_fee_sum: uint64 = uint64(npc_result.conds.reserve_fee)
 
         if fees < assert_fee_sum:
-            return (
-                None,
-                MempoolInclusionStatus.FAILED,
-                Err.RESERVE_FEE_CONDITION_FAILED,
-            )
+            return None, MempoolInclusionStatus.FAILED, Err.RESERVE_FEE_CONDITION_FAILED, None, []
 
         if cost == 0:
-            return None, MempoolInclusionStatus.FAILED, Err.UNKNOWN
+            return None, MempoolInclusionStatus.FAILED, Err.UNKNOWN, None, []
 
         fees_per_cost: float = fees / cost
         # If pool is at capacity check the fee, if not then accept even without the fee
         if self.mempool.at_full_capacity(cost):
             if fees_per_cost < self.nonzero_fee_minimum_fpc:
-                return None, MempoolInclusionStatus.FAILED, Err.INVALID_FEE_TOO_CLOSE_TO_ZERO
+                return None, MempoolInclusionStatus.FAILED, Err.INVALID_FEE_TOO_CLOSE_TO_ZERO, None, []
             if fees_per_cost <= self.mempool.get_min_fee_rate(cost):
-                return None, MempoolInclusionStatus.FAILED, Err.INVALID_FEE_LOW_FEE
+                return None, MempoolInclusionStatus.FAILED, Err.INVALID_FEE_LOW_FEE, None, []
         # Check removals against UnspentDB + DiffStore + Mempool + SpendBundle
         # Use this information later when constructing a block
         fail_reason, conflicts = await self.check_removals(removal_record_dict)
-        # If there is a mempool conflict check if this spendbundle has a higher fee per cost than all others
+        # If there is a mempool conflict check if this spend bundle has a higher fee per cost than all others
         conflicting_pool_items: Dict[bytes32, MempoolItem] = {}
-        if fail_reason is Err.MEMPOOL_CONFLICT:
-            for conflicting in conflicts:
-                sb: MempoolItem = self.mempool.removals[conflicting.name()]
-                conflicting_pool_items[sb.name] = sb
-            if not self.can_replace(conflicting_pool_items, removal_record_dict, fees, fees_per_cost):
-                potential = MempoolItem(
-                    new_spend, uint64(fees), npc_result, cost, spend_name, additions, removals, program
-                )
-                self.potential_cache.add(potential)
-                return (
-                    uint64(cost),
-                    MempoolInclusionStatus.PENDING,
-                    Err.MEMPOOL_CONFLICT,
-                )
 
-        elif fail_reason:
-            return None, MempoolInclusionStatus.FAILED, fail_reason
+        if fail_reason is not None and fail_reason is not Err.MEMPOOL_CONFLICT:
+            return None, MempoolInclusionStatus.FAILED, fail_reason, None, []
```

Comment:
so, I take it you deliberately continue here in the case of `MEMPOOL_CONFLICT`. Can you comment on that?
The code below used to not be run under that condition, but now is.

Reply:
OK, added comment